modules/standard/Path.chpl

```diff
@@ -110,4 +136,51 @@ proc file.realPath(): string {
   return ret;
 }
 
+/* Split name into a tuple that is equivalent to (:proc:`dirname`,
+   :proc:`basename`).  The second part of the tuple will never contain a slash.
+   Examples:
+
+   `splitPath("foo/bar")` will yield `("foo", "bar")`
+
+   `splitPath("bar")` will yield `("", "bar")`
+
+   `splitPath("foo/")` will yield `("foo", "")`
+
+   `splitPath("")` will yield `("", "")`
+
+   `splitPath("/")` will yield `("/", "")`
```

Comment:
I think it would be nice if this documentation comment indicated that if

    dirname, basename = split(path)

then

  dirname + "/" + basename

is the original path, with the exception that a path of "/" would become "//" in this way.